### ISSUE TITLE
Improve Queue Card Accessibility

### ISSUE THREAD

**Issue by bhuvy2** [illinois/queue#244]
**Problem**
Currently navigating Queue Cards are opaque. Landmarks and sections should be introduced so navigating a card with a screen reader makes sense.
Below is an example image with firefox's Accessibility monitor

<img width="960" alt="card_accessibility" src="https://user-images.githubusercontent.com/6509868/55015296-93946c80-4fba-11e9-9770-3da0c1c1c19e.PNG">

**Acceptance Criteria**

* The highlighted section attribute should have a top level name like card-container
* The immediate child should actually be the card
* The text container containing `CS 101` should have a title like `Queue Title`
* The node containing `Lab & Discussion` Should have a `Queue Type` Title
* The node containing location should be split up into two nodes, one with the location with the title `Location` and another with `Number of questions` containing number of questions.

Naturally if anyone wants to alter the actual English wording of anything I proposed, feel free to.
Relates to #135
